src/main/webapp/WEB-INF/views/cars/create.jsp

```diff
@@ -0,0 +1,24 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: Dastya
+  Date: 31.01.2023
+  Time: 14:24
+  To change this template use File | Settings | File Templates.
+--%>
```

Comment:
![image](https://user-images.githubusercontent.com/117590814/215799846-00105a97-9f88-4e72-8f03-b1fb85c3fc6c.png)
same in other jsp files